libs/npm/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "fine-uploader": "5.14.5"
+  }
+}
```

Comment:
I open another PR with rules for bower instead of npm (npm could follow later, but currently bower fits better for client-side stuff)